### ISSUE TITLE
Mapping stat bug?

### ISSUE THREAD

**Issue by PhrenoVermouth** [smithlabcode/abismal#4]
95.7252 + 43.6224 > 100
but coincidently 56.3776 + 43.6224 = 100
A bug for stat?

```

pairs:
    total_read_pairs: 6242210
    mapped:
        num_mapped: 5975368
        percent_mapped: 95.7252
        num_unique: 3519207
        percent_unique: 56.3776
        ambiguous: 2456161
    unmapped: 2723003
    percent_unmapped: 43.6224
    translocations: 25721
    percent_translocations: 0.41205
mate1:
    total_reads: 2723003
    mapped:
        num_mapped: 2479941
        percent_mapped: 91.0738
        num_unique: 58582
        percent_unique: 2.15137
        ambiguous: 2421359
    unmapped: 243062
    skipped: 0
mate2:
    total_reads: 2723003
    mapped:
        num_mapped: 2479941
        percent_mapped: 91.0738
        num_unique: 70986
        percent_unique: 2.6069
        ambiguous: 2408955
    unmapped: 243062
    skipped: 0
```

**Comment by andrewdavidsmith** [comment]
@PhrenoVermouth Can you specify which release that is?

**Comment by guilhermesena1** [comment]
Hello,

Thank yo for reporting this bug, you are absolutely right that we are reporting unmapped as total - unique in the paired-end map stats. I have pushed a fix for this in commit 25e7288 , in the coming 2 days or so we will be uploading a new release that will contain this fix. Once again we appreciate you reaching out about it!

**Comment by PhrenoVermouth** [comment]
Nice! :)
